uberfire-extensions/uberfire-security/uberfire-servlet-security/src/main/java/org/uberfire/ext/security/server/BasicAuthSecurityFilter.java

```diff
@@ -83,7 +90,7 @@ public void doFilter(final ServletRequest _request,
         } finally {
             // invalidate session only when it did not exists before this request
             // and was created as part of this request
```

Comment:
This comment no longer applies probably? Now it has to be specifically defined to be invalidated, right?

Reply:
@MarianMacik correct, I adjusted the comment to better reflect that.